src/backend/opencl/blas_clblas.cpp

```diff
@@ -7,6 +7,8 @@
  * http://arrayfire.com/licenses/BSD-3-Clause
  ********************************************************/
 
+#if defined(USE_CLBLAS)
```

Comment:
Yeah, I would really like to remove those variables. It is not the recommended to use that approach in CMake. I think for now you should use what @9prady9 suggested.